frontend/app/src/store/balances/blockchain-balances.ts

```diff
@@ -277,29 +324,25 @@ export const useBlockchainBalancesStore = defineStore(
 
     const adjustBlockchainPrices = async () => {
       const updateTotalsPrices = (
-        state: Ref<AssetBalances>,
-        update: (payload: AssetBalances) => any
+        state: Ref<Record<Blockchain, AssetBalances>>
       ) => {
-        const clonedState = cloneDeep(get(state));
-        for (const asset in clonedState) {
-          const assetPrice = get(prices)[asset];
-          if (!assetPrice) {
-            continue;
+        for (const chain in state) {
+          const balances = get(state)[chain as Blockchain];
+          for (const asset in balances) {
+            const assetPrice = get(prices)[asset];
+            if (!assetPrice) {
+              continue;
+            }
+            state.value[chain as Blockchain][asset] = {
```

Comment:
Same as the previous comment please don't update the `.value` properties directly